src/constants/actions.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // action types
+  TRANSITION: 'ACTION_TYPE/TRANSITION',
+  METADATA: 'ACTION_TYPE/METADATA',
+  PROGRESS: 'ACTION_TYPE/PROGRESS',
+  // transition actions
+  REJECT: 'TRANSITION/REJECT',
```

Comment:
The former is right. Just transition, metadata, and progress. (New one will be state_transition, metadata_change, progress_change)